xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-sources/src/test/java/org/xwiki/notifications/sources/internal/UsersParameterHandlerTest.java

```diff
@@ -119,6 +124,46 @@ public void handlerUsersParameter()
         assertEquals(expectedParameters, notificationParameters);
     }
 
+    @Test
+    public void handlerUsersParameterWhenOnlyTheUserParameterIsGiven() throws NotificationException
```

Comment:
note that the best practice for test is now to not use `public`.

Reply:
ok, updated.